api-get-object.go

```diff
@@ -26,6 +26,30 @@ import (
 	"time"
 )
 
+// GetSecuredObject returns a readable object
+func (c Client) GetSecuredObject(bucketName, objectName string, securedObject SecureData) (io.Reader, error) {
+
```

Comment:
We shouldn't call this `securedObject` as it confuses the user with actual object data --> `encryptionMaterials`